platform/engineering/design-docs/micro-frontend-architecture.md

```diff
@@ -0,0 +1,255 @@
+# Micro Front end Architecture
+
+**Author(s):** Chris Valarida (chris.valarida@adhocteam.us)  
+**Last Updated:** February XX, 2020  
+**Status:** **Draft** | In Review | Approved  
+**Approvers:** Andrew Gunsch, Rian Fowler, Wyatt Walter
+
+<!-- Should the approvers include anybody from DEPO? -->
+
+
+## Overview
+
+### Objective
+To establish a micro front end architecture which enables application teams to
```

Comment:
careful here: this objective is a little bit "solution in search of a problem". what are the problems we're solving with the MFE arch? why are we doing it?

(you do address this below in "Problems", but probably worth reframing the objective around that)

Reply:
Agreed on all points!